iele.md

```diff
@@ -1048,24 +995,23 @@ The `JUMP*` family of operations affect the current program counter.
 
     syntax NullVoidOp ::= JUMP ( Int )
  // ----------------------------------
-    rule <k> JUMP(LABEL) => . ... </k> <jumpTable> ... LABEL |-> DEST </jumpTable> <pc> _ => DEST </pc>
+    rule <k> JUMP(LABEL) ~> _:Ops => CODE ... </k> <fid> FUNC </fid> <funcId> FUNC </funcId> <jumpTable> ... LABEL |-> CODE </jumpTable>
```

Comment:
Same comment as the other jumpTable lookup - maybe explicitly write `<function>... ...</function>` rather than relying on context abstraction.